lib/src/ast/sass/interface/reference.dart

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:meta/meta.dart';
+import 'package:source_span/source_span.dart';
+
+import '../node.dart';
+
+/// A common interface any node that references a Sass member.
+///
+/// {@category AST}
+@sealed
+abstract class SassReference extends SassNode {
+  /// The namespace of the member being referenced, or `null` if it's referenced
+  /// without a namespace.
+  String? get namespace;
+
+  /// The name of the member being referenced, with underscores converted to
+  /// hyphens.
+  ///
+  /// For [VariableExpression]s and [IncludeRule]s, this will never be null.
+  /// For [FunctionExpression]s, this will be null if the actual name is an
+  /// interpolation (in which case this is a plain CSS function, not a reference
+  /// to a Sass function).
```

Comment:
I think this indicates that we should split out a `CssFunctionExpression` from `FunctionExpression`. Not only will it make this easier for users of the AST to handle consistently, it'll make it clear when a function is or is not `-`-normalized.